### DISCUSSION TITLE
State update of Osram Smart+ plug (AB3257001NJ)

### DISCUSSION THREAD

**Discussion by avanlievenoogen** [Koenkk/zigbee2mqtt#6046] [Devices]
I'm using Zigbee2mqtt (1.17.0) with a conbee II and home assistant (2021.1.5)

I'm trying to get an update of the OSRAM Smart+ plug (AB3257001NJ) when the button on the plug is pressed.
I don't see any evidence in the logging that any state updates are pushed from the device. Only when I query the device via a "get" MQTT command, the state is updated.

Question:
- Do you have a suggestion why the state is not pushed from the device?
- If it is supposed to work like this, what is the best way to poll the different plugs I have? (automation in HA, configuration in Zigbee2mqtt,...)

**Comment by Mark-Sellwood** [comment]
If I look in the logs on the FrontEnd I see the below when I press the button on the socket:-
Info MQTT publish: topic 'zigbee2mqtt/Osram 01', payload '{"linkquality":135,"state":"ON","update":{"state":"idle"},"update_available":false}'
Info MQTT publish: topic 'zigbee2mqtt/Osram 01', payload '{"linkquality":132,"state":"OFF","update":{"state":"idle"},"update_available":false}'

**Reply by avanlievenoogen** [reply]
hmm, that is what I would expect. However, I don't see that. I also see nothing in the underlying Zigbee-herdsman debug logging. Could it be Conbee II related?
I do receive an update after sending the get command so I would think the infrastructure is working

**Reply by Mark-Sellwood** [reply]
I'm new to ZigBee, it could be Conbee.
I'm using a slae.sh CC2652RB stick with Domoticz
